src/api-service/__app__/node/__init__.py

```diff
@@ -73,6 +73,8 @@ def delete(req: func.HttpRequest) -> func.HttpResponse:
             context=request.machine_id,
         )
 
+    NodeMessage.clear_messages(node.agent_id)
```

Comment:
IMO, these should probably be cleared at the Node.delete() level, such that it's called when the service finalizes the delete.

https://github.com/microsoft/onefuzz/blob/46e84545696b1c69466ebaa1941648d085b98c95/src/api-service/__app__/onefuzzlib/pools.py#L318-L320